esmvaltool/utils/batch-jobs/generate.py

```diff
@@ -0,0 +1,86 @@
+import esmvaltool
+import esmvalcore
+import os
+import glob
+
+from pathlib import Path
+
+env = 'coretool26rc4'
+mail = 'saskia.loosveldt@bsc.es'
```

Comment:
Can I set the mail to `None` ?

Reply:
Yes, you should. At DKRZ, it is not necessary to specify the email address in the batch script. Having `#SBATCH --mail-type=<something>` is enough to get emails at the address that is linked to your DKRZ account.